org.kde.katomic.json

```diff
@@ -1,13 +1,13 @@
 {
     "id": "org.kde.katomic",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.11",
+    "runtime-version": "5.12",
     "sdk": "org.kde.Sdk",
     "command": "katomic",
     "rename-icon": "katomic",
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
```

Comment:
Can you please explain in the commit log this change? All the others changes are either explained or self explanatory but this one is a bit different

Reply:
`--fallback-x11` means that Flatpak will try to launch with Wayland firstly and only fall back on x11 if Wayland didn't work out all right.

Reply:
Sounds good, can you please update the commit log to say that?

Reply:
Done :)